projects/plugins/boost/app/assets/src/js/pages/cache-debug-log/cache-debug-log.tsx

```diff
@@ -0,0 +1,47 @@
+import { __ } from '@wordpress/i18n';
+import Header from '$layout/header/header';
+import { useDataSync } from '@automattic/jetpack-react-data-sync-client';
+import { z } from 'zod';
+import Footer from '$layout/footer/footer';
+import styles from './cache-debug-log.module.scss';
+import classNames from 'classnames';
+import { Button } from '@automattic/jetpack-components';
+
+const CacheDebugLog = () => {
+	const [ { data: debugLog } ] = useDataSync( 'jetpack_boost_ds', 'cache_debug_log', z.string(), {
+		query: {
+			// Keep refreshing the logs every 10 seconds
+			refetchInterval: 10000,
+		},
+	} );
+
+	const copyToClipboard = () => {
+		navigator.clipboard.writeText( debugLog as string );
+	};
+
+	return (
+		<div id="jb-dashboard" className="jb-dashboard jb-dashboard--main">
+			<Header subPageTitle={ __( 'Cache Log Viewer', 'jetpack-boost' ) } />
+			<div className={ classNames( 'jb-section jb-section--main', styles.section ) }>
+				<div className="jb-container">
+					<header className={ styles.header }>
+						<h3>{ __( 'Jetpack Boost Cache Log Viewer', 'jetpack-boost' ) }</h3>
+						<Button
+							className={ styles[ 'copy-button' ] }
+							onClick={ copyToClipboard }
```

Comment:
copyToClipboard throws an error in my browser: 
>  Cannot read properties of undefined (reading 'writeText')

Reply:
I was going with navigator.clipboard, which seems to not be usable in insecure context. Seems like RNA alreadys has a CopyToClipboard button that comes with an icon feedback about the copy. We can remove the icon(to follow design) but I think it's better if we keep it.


https://github.com/Automattic/jetpack/assets/3737780/82922fdb-ba96-4414-b3e8-fc56023c997e